From 707406e2510437c188a1a1c51293e9cb8fa25bc4 Mon Sep 17 00:00:00 2001
From: Oliver Neukum <oneukum@suse.com>
Date: Mon, 30 May 2022 14:36:14 +0200
Subject: [PATCH] hide appended member supports_dynamic_smps_6ghz
Patch-mainline: Never (kABI fixup)
References: git-fixes

struct ath11k_hw_params is just extended 1 byte and cannot be embedded
or dynamically allocated

Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 drivers/net/wireless/ath/ath11k/hw.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/wireless/ath/ath11k/hw.h b/drivers/net/wireless/ath/ath11k/hw.h
index 22b3f36b0..ccb324b69 100644
--- a/drivers/net/wireless/ath/ath11k/hw.h
+++ b/drivers/net/wireless/ath/ath11k/hw.h
@@ -163,7 +163,9 @@ struct ath11k_hw_params {
 	bool supports_suspend;
 	u32 hal_desc_sz;
 	bool fix_l1ss;
+#ifndef __GENKSYMS__
 	bool supports_dynamic_smps_6ghz;
+#endif
 };
 
 struct ath11k_hw_ops {
-- 
2.35.3

